CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.13](https://github.com/brave/brave-ios/releases/tag/v1.13)
+ 
+ - Implement Brave Rewards and Ads. ([#1920](https://github.com/brave/brave-ios/issues/1920))
+ - Added new preview menu on iOS13. ([#1499](https://github.com/brave/brave-ios/issues/1499))
+ - Added bookmarklet support. ([#1119](https://github.com/brave/brave-ios/issues/1119))
+ - Updated text size and colours on new tab stats to match desktop. ([#1641](https://github.com/brave/brave-ios/issues/1641))
+ - Updated StartPage icon. ([#1598](https://github.com/brave/brave-ios/issues/1598))
+ - Fixed wrong device name being shown when tapped on sync devices. ([#1587](https://github.com/brave/brave-ios/issues/1587))
+ - Fixed readablity of website/tab names on dark mode. ([#1551](https://github.com/brave/brave-ios/issues/1551))
```

Comment:
Change `readablity` --> `readability`. Should probably change `on` --> `in`.  Either of these would work:

* Fixed readability of websites/tab names in dark mode.
* Fixed readability of websites/tab names when using dark mode.